mrcfile/mrcobject.py

```diff
@@ -157,58 +159,100 @@ def _create_default_header(self):
         # (this can also be achieved by assigning a 3-tuple to header.cellb
         # directly but using the sub-fields individually is easier to read and
         # understand)
-        
+
         # Standard axes: columns = X, rows = Y, sections = Z
         header.mapc = 1
         header.mapr = 2
         header.maps = 3
-        
+
         time = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
         header.label[0] = '{0:40s}{1:>39s} '.format('Created by mrcfile.py',
                                                     time)
         header.nlabl = 1
-        
+
         self.reset_header_stats()
-    
+
     @property
     def header(self):
         """Get the header as a :class:`numpy record array <numpy.recarray>`."""
         return self._header
-    
+
     @property
     def extended_header(self):
         """Get the extended header as a :class:`numpy array <numpy.ndarray>`.
-        
-        If this :class:`MrcObject` was read from a file and the extended header
-        type was recognised, its dtype will be set appropriately. (Currently
-        the only supported types are ``'FEI1'`` and ``'FEI2'``.) Otherwise, the
-        dtype will be void (raw data, dtype ``'V'``). If the actual data type
+
+        The dtype will be void (raw data, dtype ``'V'``). If the actual data type
         of the extended header is known, the dtype of the array can be changed
-        to match.
-        
+        to match. For supported types (e.g. ``'FEI1'`` and ``'FEI2'`), the
+        indexed part of the extended header (excluding any zero padding) can be
+        accessed using :meth:`indexed_extended_header`
+
         The extended header may be modified in place. To replace it completely,
         call :meth:`set_extended_header`.
         """
         return self._extended_header
-    
+
+    @property
+    def indexed_extended_header(self):
+        """Get the indexed part of the extended header as a
+        :class:`numpy array <numpy.ndarray>` with the appropriate dtype set.
```

Comment:
```suggestion
        :class:`numpy array <numpy.ndarray>` with the appropriate dtype set.

```

It's nice for the docstrings to have a title separated from the more detailed description. Ideally the title would be only one line. I think a two-line one is fine here, but it still needs a blank line to separate it from the rest.